app/models/epp/domain.rb

```diff
@@ -475,6 +478,20 @@ def update(frame, current_user, verify = true)
 
     same_registrant_as_current = (registrant.code == frame.css('registrant').text)
 
+    if !same_registrant_as_current && disputed?
+      disputed_pw = frame.css('disputed > pw').text
+      if disputed_pw.blank?
+        add_epp_error('2304', nil, nil, 'Required parameter missing; disputed' \
+        'pw element required for dispute domains')
+      else
+        dispute = Dispute.active.find_by(domain_name: name, password: disputed_pw)
+        if dispute.nil?
+          add_epp_error('2202', nil, nil, 'Invalid authorization information; '\
+          'invalid disputed>pw value')
```

Comment:
Suppose that's a typo, don't we need underscore here? LIke 'disputed_pw'

Reply:
Actually this was intentional as well, as same structure was used before for reserved domains / other auths as well. EPP has an object `disputed` which has a `pw` field.